src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/test_storage_account_scenarios.py

```diff
@@ -121,6 +120,18 @@ def test_create_storage_account(self, resource_group, location):
         self.cmd('storage account check-name --name {}'.format(name),
                  checks=JMESPathCheck('nameAvailable', True))
 
+    @ResourceGroupPreparer(parameter_name_for_location='location', location='southcentralus')
```

Comment:
Does this not need `api_version_constraint`?

Reply:
will update.